src/main/java/core/basesyntax/service/DataHandlerImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;

```

Comment:
implementations must be in the impl package
```suggestion
package core.basesyntax.service.impl;

```